examples/README.md

```diff
@@ -1,8 +1,31 @@
-# Custom Script Examples
+# Instrumental Daemon Examples
+
+Here you can find examples of custom plugin scripts and other bits for Instrumental Daemon.
+
+## Plugin Script Examples
 
 The following scripts are examples of how to monitor other parts of your system not natively supported by `instrumentald`.
 
-* [MySQL](mysql/README.md)
-* [Docker](docker/README.md)
-* [Mongo](mongo/README.md)
-* [Redis](redis/README.md)
+#### Log Size
+
+The [log size](plugin_scripts/log_size.sh) script simply monitors the size of logs in /var/log, so you can see if a log is failing to rotate or growing out of control.  It demonstrates using an executable shell script as a plugin script.
+
+#### ISS Monitor
+
+The [ISS Monitor](plugin_scripts/iss_monitor.rb) script monitors the distance between my current location and the international space station (or, more exactly, a point directly UNDER the international space station).  You can use environment variables to track the distance between your location and the ISS.
+
+#### MySQL
+
+The [MySQL](plugin_scripts/mysql.rb) script collects the following metrics:
+
+* Queries - `Queries`
+* Bytes Sent  - `Bytes_sent`
+* Bytes Received - `Bytes_received`
+* Connections - `Connections`
+* Slow queries - `Slow_queries`
+
+Additionally, it estimates the number received per second ( `Queries_per_second`, etc. ).
+
+MySQL monitoring functionality is provided by default, but this script demonstrates advanced functionality, including receiving previous script output via STDIN and other advanced features.
```

Comment:
Yes, we should definitely add a link to that section from here when we add that section.